### ISSUE TITLE
 No such property: COMPILE_ONLY_CONFIGURATION_NAME for class: org.gradle.api.plugins.JavaPlugin

### ISSUE THREAD

**Issue by timotei** [etiennestuder/gradle-plugindev-plugin#35]
It seems that with latest version I get this:

```

Caused by: org.gradle.api.internal.plugins.PluginApplicationException: Failed to apply plugin [id 'nu.studer.plugindev']
        at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction.applyPlugin(DefaultObjectConfigurationAction.java:117)
        at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction.access$200(DefaultObjectConfigurationAction.java:36)
        at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction$3.run(DefaultObjectConfigurationAction.java:80)
        at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction.execute(DefaultObjectConfigurationAction.java:131)
        at org.gradle.api.internal.project.AbstractPluginAware.apply(AbstractPluginAware.java:37)
        at org.gradle.api.Project$apply$0.call(Unknown Source)
        at org.gradle.api.internal.project.ProjectScript.apply(ProjectScript.groovy:34)
        at org.gradle.api.Script$apply$0.callCurrent(Unknown Source)
        at build_an60zvzl0fynwc5b0jufsy5gw.run(E:\src\p\gradle-editbin-plugin\build.gradle:19)
        at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:52)
        ... 42 more
Caused by: groovy.lang.MissingPropertyException: No such property: COMPILE_ONLY_CONFIGURATION_NAME for class: org.gradle.api.plugins.JavaPlugin
        at nu.studer.gradle.plugindev.PluginDevPlugin.apply(PluginDevPlugin.groovy:106)
        at nu.studer.gradle.plugindev.PluginDevPlugin.apply(PluginDevPlugin.groovy)
        at org.gradle.api.internal.plugins.DefaultPluginContainer.providePlugin(DefaultPluginContainer.java:188)
        at org.gradle.api.internal.plugins.DefaultPluginContainer.addPluginInternal(DefaultPluginContainer.java:137)
        at org.gradle.api.internal.plugins.DefaultPluginContainer.apply(DefaultPluginContainer.java:103)
        at org.gradle.api.internal.plugins.DefaultObjectConfigurationAction.applyPlugin(DefaultObjectConfigurationAction.java:115)
        ... 51 more
```

But this didn't happen with 1.0.3

**Comment by etiennestuder** [comment]
What version of Gradle are you on? This constant is available since Gradle [2.12](https://docs.gradle.org/2.12/javadoc/org/gradle/api/plugins/JavaPlugin.html#COMPILE_ONLY_CONFIGURATION_NAME).

**Comment by etiennestuder** [comment]
I've released 1.0.12 which provides backward compatibility down to Gradle 3.5.1.

**Comment by timotei** [comment]
Oh, it seems a super old version :) 2.2.1. With newer, works fine :)

Thank you!
